packages/ag-charts-community/src/scale/continuousScale.ts

```diff
@@ -80,6 +82,11 @@ export abstract class ContinuousScale<D extends number | Date, I = number> imple
         );
     }
 
+    /** Override point for special convert() handling by sub-classes. */
+    protected specialConvert(x: D, _domainStart: D, _domainEnd: D) {
```

Comment:
is this new? I'm not seeing it used anywhere (maybe I missed it)

Reply:
Good spot - removed, this was a hang-over from my battle with `LogScale` :P